### ISSUE TITLE
HmiP-FBL chaotic behaviour in automations 

### ISSUE THREAD

**Issue by fr3akst3r** [marcsowen/homebridge-homematicip#279]
Hi there,

I am experiencing unexpected behaviour of multiple HmiP-FBLs wenn using f.e. a HomeKit automation for closing all jalousies after leaving home. 
Behaviour is especially seen when to be closed jalousies are completely open. They just move a bit down and go up again. 
HmIP-BROLLs are behaving just fine.

**Comment by marcsowen** [comment]
Could you enable the debug log output and send the logs while the automation is closing the jalousies? I don't own this actuator so I cannot really reproduce it myself unfortunately.

**Comment by fr3akst3r** [comment]
After having found out how to do that I will reply with a log. Thanks for your reply!

**Comment by sidthekid87** [comment]
Hi there,

I have the same issue. I guess, it occurs because of the setting of the slats position. I think the actuator is handling these sequential. You see below from the log, that the order is not always the same (sometimes the slats position is set first, sometimes not).

`[2/21/2023, 6:00:36 PM] [HomematicIP] Setting target shutter position for Storen rechts to 0 %
[2/21/2023, 6:00:36 PM] [HomematicIP] Setting target horizontal slats position for Storen links to 0°
[2/21/2023, 6:00:36 PM] [HomematicIP] Setting target horizontal slats position for Storen rechts to 0°
[2/21/2023, 6:00:36 PM] [HomematicIP] Setting target shutter position for Storen links to 0 %`

I set the actuator in the Homematic IP app to roller mode. Is there a possibility to exclude the slats feature in the Homebridge config as well?

**Comment by fr3akst3r** [comment]
Also still having the issue. 
setting the actuator in BROLL mode is kind of a pity, as you just lose the slat setting function, don’t you?

**Comment by sidthekid87** [comment]
Yes, you are right about that. But at least my automations would maybe work again.... The best would be: First set the opening level, then wait until this height is reached, and then set the tilt.
Not sure, if this would be an Homekit thing or a plugin thing...

**Comment by sidthekid87** [comment]
It really seems to be the slats positioning command which causes the chaos.
Manual opening / closing over Homekit result in following log entries:
<img width="439" alt="image" src="https://user-images.githubusercontent.com/39822489/220446278-bdf99020-fdec-4bc0-84eb-0b13d4a1122f.png">

Closing over a scene in Homekit results in this commands:
<img width="467" alt="image" src="https://user-images.githubusercontent.com/39822489/220446439-788234e3-70b1-4757-ac67-f9a0762ce055.png">

**Comment by marcsowen** [comment]
HomeKit scenes set the blind height and slat angle separately. The plugin used the old height for the slat command, which could make the blind open again. This is now fixed.

Slat controls are also hidden automatically when the actuator is configured in roller mode. The fix will be available with the next release.
